src/frontend/Sider/Personoversikt/Behandlingsoversikt/OpprettNyBehandling/OpprettOrdinærBehandling.tsx

```diff
@@ -101,11 +140,23 @@ const OpprettOrdinærBehandling: React.FC<Props> = ({
         if (harVerdi(value)) {
             settÅrsak(value as BehandlingÅrsak);
             settFeilmelding(undefined);
+            settKilde(undefined);
+            settEndringer(undefined);
+            settBeskrivelse(undefined);
         } else {
             settÅrsak(undefined);
         }
     };
 
+    const endreKilde = (event: React.ChangeEvent<HTMLSelectElement>) => {
+        const value = event.target.value;
+        if (harVerdi(value)) {
+            settKilde(value as NyeOpplysningerKilde);
+            settFeilmelding(undefined);
+        } else {
+            settKilde(undefined);
+        }
+    };
```

Comment:
Tror en av disse burde fungere hvis man fjerner `value` fra "velg"-option. Siden man må sette denne verdien dersom årsaken er nye opplysninger så gir det mening at den ikke kan settes tilbake til velg?

```suggestion
    const endreKilde = (kilde: NyeOpplysningerKilde) => {
            settKilde(kilde);
            settFeilmelding(undefined);
        } 
    };
```
+ endring i onChange: 
```js
onChange={(e) => e as NyeOpplysningerKilde}
```
Eller
```suggestion
    const endreKilde = (event: React.ChangeEvent<HTMLSelectElement>) => {
            settKilde(event.target.value as NyeOpplysningerKilde);
            settFeilmelding(undefined);
        } 
    };
```

Reply:
Ser at aksel bruker "" som value så det er kanskje anbefalt måte. Da kunne man evt bare gjort:

```suggestion
    const endreKilde = (event: React.ChangeEvent<HTMLSelectElement>) => {
        const value = event.target.value;
        settKilde(harVerdi() ? value as NyeOpplysningerKilde : undefined)
        settFeilmelding(undefined);
    };
```